src/oadp_resources/oadp/datamover/volume_snapshot_restore.py

```diff
@@ -1,6 +1,49 @@
 from ocp_resources.resource import NamespacedResource
+
+from oadp_constants.oadp.datamover.volume_snapshot_restore import VolumeSnapshotRestorePhase
 from oadp_constants.resources import ApiGroups
 
 
-class VolumeSnapshotLocation(NamespacedResource):
+class VolumeSnapshotRestore(NamespacedResource):
     api_group = ApiGroups.DATAMOVER_OADP_API_GROUP.value
+
+    def is_vsr_done(self):
+        """
+        Check is VSR process is done
+        @return: True if the VSR process is not running; False otherwise
+        """
+        return self.instance.status and self.instance.status.phase != \
+               VolumeSnapshotRestorePhase.SnapMoverRestorePhaseInProgress.value
+
+    @classmethod
+    def get_vsrs_by_restore_name(cls, restore_name):
```

Comment:
u can have same vsr with same name ?

Reply:
no, but can have multiple VSRs with the same restore name in the labels

Reply:
again, not sure it it the right place.
@mperetzred  @tareqalayan